### PR TITLE
ADD: nbparameterise

### PR DESCRIPTION
**Pull request by kmuehlbauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
Turns out that `astsearch` and `astcheck` have to be added before this one.

**Comment by kmuehlbauer** [comment]
@takluyver This adds `astsearch` and `nbparameterise`. It would be great, if you can have a look at both recipes, especially at the build/script-section and requirements. It seems everything is working, but you might have suggestions for improvement. And finally, would you mind me adding you as maintainer of these two packages?

@conda-forge/staged-recipes Please have have a look as well.

**Comment by takluyver** [comment]
You possibly want the `--no-build-isolation` flag for pip install, to stop pip from downloading flit itself to put in an isolated build environment - conda manages the build environment itself. But I don't know how strict conda-forge is about this in general - I'm sure there are other recipes that let pip create its own build environment.

I don't mind being listed as a maintainer, but don't expect me to do much on it. I'm on so many conda-forge feedstock repositories already, I tend to ignore anything coming from there.

**Comment by kmuehlbauer** [comment]
> You possibly want the `--no-build-isolation` flag for pip install, to stop pip from downloading flit itself to put in an isolated build environment - conda manages the build environment itself. But I don't know how strict conda-forge is about this in general - I'm sure there are other recipes that let pip create its own build environment.

@conda-forge/staged-recipes Are there any guidelines on that? A quick search didn't brought up anything specific.

File: recipes/astsearch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5db56de6ba7ecce1f5e5ba1fc6bf52a9bcf25ce330dc0decf001eb519e90de96
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/astsearch/meta.yaml:13]: Any reason why this isn't `noarch: python`?

Comment (kmuehlbauer) [reply recipes/astsearch/meta.yaml:13]: @chrisburr It creates a console script. It was complaining on windows about not being able to be noarch.

I therefore removed the noarch.

Comment (chrisburr) [reply recipes/astsearch/meta.yaml:13]: See https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

**Pull request merged**